pvlib/iotools/psm3.py

```diff
@@ -9,7 +9,8 @@
 import pandas as pd
 from json import JSONDecodeError
 
-URL = "http://developer.nrel.gov/api/solar/nsrdb_psm3_download.csv"
+PSM_URL = "http://developer.nrel.gov/api/solar/nsrdb_psm3_download.csv"
+TMY_URL = "http://developer.nrel.gov/api/nsrdb_api/solar/nsrdb_psm3_tmy_download.csv"  # noqa
```

Comment:
Suggestion, not blocker. There's some repeat text in the URLs, perhaps add `NREL_API`:
```python
NREL_API = 'http://developer.nrel.gov/api/'
PSM_URL = NREL_API + "solar/nsrdb_psm3_download.csv"
TMY_URL = NREL_API + "nsrdb_api/solar/nsrdb_psm3_tmy_download.csv"
```
This also results in shorter lines without using `#noqa`.

Also can you double check the URL scheme? I'm a little surprised they are not using HTTPS for endpoints that transmit credentials

Reply:
Good suggestions.  I've split out the domain from the endpoint paths as you suggest.

The example python snippets in the docs use HTTP but it looks like HTTP and HTTPS both work. I've switched to HTTPS.